### PR TITLE
distil-finetuning in run_squad

### PR DESCRIPTION
**Pull request by VictorSanh**
- Add the option for double loss: fine-tuning + distillation from a larger squad-finetune model.
- Fix `inputs` for `DistilBERT` (also see fix in  `run_glue.py` 702f589848baba97ea4897aa3f0bb937e1ec3bcf)

### CODE REVIEW

**Comment by VictorSanh** [comment]
cf https://github.com/huggingface/transformers/issues/1193#issuecomment-534740929

**Comment by thomwolf** [comment]
Ok, as discussed let's copy this script to the `examples/distillation` folder and keep `run_squad` barebone for now as it's going to evolve in the short term.

**Comment by julien-c** [comment]
no squash @VictorSanh? 😬

File: examples/distillation/run_squad_w_distillation.py [hunk 1]
```diff
@@ -0,0 +1,585 @@
...
+                                  BertForQuestionAnswering, BertTokenizer,
+                                  XLMConfig, XLMForQuestionAnswering,
+                                  XLMTokenizer, XLNetConfig,
+                                  XLNetForQuestionAnswering,
+                                  XLNetTokenizer,
+                                  DistilBertConfig, DistilBertForQuestionAnswering, DistilBertTokenizer)
+
+from transformers import AdamW, WarmupLinearSchedule
+
+from ..utils_squad import (read_squad_examples, convert_examples_to_features,
```
Comment (UMaa69) [inline examples/distillation/run_squad_w_distillation.py:47]: importing from  utils _squad  is failing
